src/summary/cards/horizontal-label-value.component.tsx

```diff
@@ -11,13 +11,23 @@ export default function HorizontalLabelValue(props: HorizontalLabelValueProps) {
         {props.label}
         {props.specialKey && <sup>{"\u002A"}</sup>}
       </label>
-      <div
-        title={props.label}
-        className={props.valueClassName || "omrs-type-body-regular"}
-        style={props.valueStyles}
-      >
-        {props.value || "\u2014"}
-      </div>
+      {props.value ? (
+        <div
+          title={props.label}
+          className={props.valueClassName || "omrs-type-body-regular"}
+          style={props.valueStyles}
+        >
+          {props.value || "\u2014"}
+        </div>
+      ) : null}
+      {props.detail ? (
+        <div
+          className={props.detailsClassName || "omrs-type-body-regular"}
+          style={props.detailsStyles}
+        >
+          {props.detail || "\u2014"}
+        </div>
+      ) : null}
```

Comment:
You can do this instead of using the ternary in this situation
```suggestion
      {props.detail && 
        <div
          className={props.detailsClassName || "omrs-type-body-regular"}
          style={props.detailsStyles}
        >
          {props.detail || "\u2014"}
        </div>
```